tests/gis_tests/inspectapp/tests.py

```diff
@@ -50,10 +51,14 @@ def test_3d_columns(self):
         output = out.getvalue()
         if connection.features.supports_geometry_field_introspection:
             self.assertIn('point = models.PointField(dim=3)', output)
+            if postgis:
```

Comment:
Could be a separate ticket but might be worth adding something to `connection.features` for this?